src/tests/ngraph_helpers/snippets_ngraph_functions/include/subgraph_simple.hpp

```diff
@@ -227,6 +227,21 @@ class BroadcastSelectFunction : public SelectFunction {
 
     PartialShape m_target_shape;
 };
+
+/// yolo-v3 subgraph to cover edge replacement.
+//                subgraph
+//                /     |
+//           subgraph  subgraph
+//                \     /
+//              eltwise node
+class EdgeReplaceFunction : public SnippetsFunctionBase {
+public:
+    explicit EdgeReplaceFunction(const std::vector<PartialShape>& inputShapes) : SnippetsFunctionBase(inputShapes) {
+        NGRAPH_CHECK(input_shapes.size() == 8, "Got invalid number of input shapes");
```

Comment:
Please note that we don't have to reproduce the entire subgraph from the real model here. It would be really help to reduce maintenance costs if you could simplify the subgraph as much as possible, but in a way that it still covers the introduced changes. 
For example, you can use minimalistic one-operation subgraphs and replace unnecessary Parameter inputs with scalar Constants. This way we won't need 8 inputs.

Reply:
Simplified.
common parent could be node that have multiple output, now use a split node instead of a big subgraph to represent it.
one block is enough to reproduce, removed two parallel blocks.
reduced input numbers with constant internally, as this issue is not related input values.
Thank you!